850.split-ambiguities/s.yaml

```diff
@@ -42,6 +42,11 @@
 - { name: scripts, wwwpart: gobolinux, setname: gobolinux-scripts }
 - { name: scripts, warning: "please classify me" }
 
+- { name: sd, wwwpart: syncwith, setname: sd-syncwith }
+- { name: sd, ruleset: exherbo, setname: sd-syncwith }
+- { name: sd, wwwpart: chmln/sd, setname: "rust:sd" }
```

Comment:
Ruleset based rules not accepted; modules prefix (`rust:`) is discouraged for utilities. App-SD is not covered. Related version rules are not removed.